app/uk/gov/hmrc/organisationsdetailsapi/views/definition.scala.txt

```diff
@@ -0,0 +1,23 @@
+@import play.api.libs.json.Json
+@()
+{
+  "scopes": [
+    {
+      "key": "assigned",
+      "name": "The scopes you have been assigned",
+      "description": "An OAuth 2.0 Bearer Token with the scopes you have been assigned"
+    }
+  ],
+    "api": {
+      "name": "Organisations Details",
+      "description": "Retrieves details information for an organisation",
+      "context": "organisations/details",
+      "versions": [
+        {
+          "version": "P1.0",
```

Comment:
@parkeradam  - do we want to make this configurable now like;

https://github.com/hmrc/individuals-employments-api/blob/master/app/uk/gov/hmrc/individualsemploymentsapi/views/definition.scala.txt#L102

https://github.com/hmrc/individuals-employments-api/blob/master/app/uk/gov/hmrc/individualsemploymentsapi/controller/DocumentationController.scala#L45-L53

Saves doing it later and means we can control the ALPHA/BETA status on deploy to individual environments?